sunpy/util/exceptions.py

```diff
@@ -55,3 +57,51 @@ class SunpyPendingDeprecationWarning(PendingDeprecationWarning, SunpyWarning):
     """
     A warning class to indicate a soon-to-be deprecated feature.
     """
+
+
+def warn_meta(msg, stacklevel=1):
```

Comment:
I think it's marginally clearer to say metadata?

```suggestion
def warn_metadata(msg, stacklevel=1):
```

Reply:
👍 there's a few minor things I need to fix anyway, so I can make this change too